Readme.md

```diff
@@ -4,109 +4,57 @@
-> The description of this example is currently under construction and may not match the code in the example. We are currently working to provide you with up-to-date content.
-
-This example illustrates one of the possible approaches to implementing an identifier field with sequential values.
+This example illustrates how to implement a business object with an identifier field with autogenerated sequential values.
 
 ![](https://raw.githubusercontent.com/DevExpress-Examples/how-to-generate-a-sequential-number-for-a-persistent-object-within-a-database-transaction-xaf-e2829/17.2.8+/media/9ecee31b-58bf-11e6-80bf-00155d62480c.png)
 
-## Scenario
-
-This is a variation of the [How to generate and assign a sequential number for a business object within a database transaction, while being a part of a successful saving process](https://www.devexpress.com/Support/Center/p/E2620) XPO example, which was specially adapted for XAF applications.
-
-In particular, for better reusability and more smooth integration with the standard XAF CRUD Controllers, all the required operations to generate sequences are managed within the base persistent class automatically when a persistent object is being saved. For more developer convenience, this solution is organized as a reusable XAF module (_GenerateUserFriendlyId.Module_). This module consists of several key parts:
+> **Note** 
+> This Readme focuses on Entity Framework Core. For information on how to achieve the same functionality with XPO, see the **Readme.md** file in the [XPO solution's folder](./CS/XPO/).
 
-* `Sequence` and `SequenceGenerator` are auxiliary classes that take the main part in generating user-friendly identifiers. Take special note that the `SequenceGenerator.Initialize` method must be called during your XAF application startup for the correct operation.
-* `UserFriendlyIdPersistentObject` is a base persistent class that subscribes to XPO's Session events and delegates calls to the core classes above. Normally, you must inherit your own business classes from this base class to get the described functionality in your project.
-* `IUserFriendlyIdDomainComponent` is a base domain component that should be implemented by all domain components that require the described functionality.
+## Implementation Details
 
-Check the original example description first for more information on the demonstrated scenarios and functionality.
+Entity Framework Core allows you to set up generation of sequential values for non-key data fields as described in the following article in the Entity Framework Core documentation: [Generated Values - Explicitly configuring value generation](https://learn.microsoft.com/en-us/ef/core/modeling/generated-properties?tabs=data-annotations#explicitly-configuring-value-generation).
 
-## Implementation Details
+Use the steps bellow to generate sequential values for a business object's property so that the property is assigned a value once the object has been saved to the database:
```

Comment:
```suggestion
Use the steps below to generate sequential values for a business object's property so that the property is assigned a value once the object has been saved to the database:
```